child_compassion/models/household.py

```diff
@@ -30,6 +30,9 @@ class Household(models.Model):
         readonly=True
     )
 
+    _sql_constraints = [('household_uniq', 'unique(household_id)',
+                        ('An Household with the same ID already exists.'))]
+
```

Comment:
I added the unique constraint but couldn't reproduce the bug (new Household is created with the same ID when updating an existing one).
The update doesn't seem to be the issue.